generic/samples/proguard/README.md

```diff
@@ -4,6 +4,8 @@ Sample project for [Ktor](https://ktor.io) packed as a JAR and minimized using [
 
 ## Running
 
+In order to minimize a JAR you need to use JDK 8 because file `lib/rt.jar` 
```

Comment:
In order to minimize a JAR file, you need to use JDK 8. This is due to the fact that `lib/rt.jar` which is required by [ProGuard], is no longer available in [JDK 9].